aries-site/src/examples/templates/FilterControls/FiltersContext.js

```diff
@@ -107,6 +117,18 @@ const useFilters = () => {
     return filterResults;
   };
 
+  const getSelected = (array1, array2) => {
+    const results = [];
+    array1.forEach(i => {
+      let inBoth;
+      array2.forEach(j => {
+        if (i === j[primaryKey]) inBoth = true;
+      });
+      if (inBoth) results.push(i);
+    });
```

Comment:
Does something like this yield the same result?

```suggestion
    const results = array1.filter(item => array2.indexOf(item) !== -1);;
```

Reply:
Not quite, we still need to compare `i` to the `primaryKey` in each of the `j` items. Happy to look more into how to simplify this function, but might feel more comfortable taking in a separate cleanup